app/main.py

```diff
@@ -16,4 +16,5 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films, marks, collection_of_coins]}
+sorted_variables = {"immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]}
```

Comment:
Define one variable `sorted variables` and form it as mentioned in the task description, e.g.:
```
sorted_variables = {
    "mutable": [...],
    "immutable": [...]
}
``` 

 Make sure that there are no `flake8` or test errors upon the next review request.